native/src/components/News.tsx

```diff
@@ -63,6 +64,16 @@ const News = ({
   const selectedNewsItem = news.find(_newsItem => _newsItem.id === newsId)
   const { t } = useTranslation('news')
 
+  const { setTitle, setContent } = useTtsPlayer()
+
+  useEffect(() => {
```

Comment:
couldn't you just pass these values to the hook instead of using a separate useEffect here. I think the lifecycle depending things should be handled in the hook

Reply:
I didn't get what exactly you want to me to do here... just pass them to the hook without useeffect?
I think in this case it will keep setting these states each time the News re-render right?